usaspending_api/accounts/models.py

```diff
@@ -105,9 +127,17 @@ def __str__(self):
         return "%s" % (self.tas_rendering_label)
 
 
-# Table #4 - Appropriation Account Balances
 class AppropriationAccountBalances(DataSourceTrackedModel):
+    """
+    Represents Treasury Account Symbol (TAS) balances for each DATA Act
+    broker submission. Each submission provides a snapshot of the most
+    recent numbers for that fiscal year. In other words, the lastest
+    submission for a fiscal year reflects the balances for the entire
+    fiscal year.
+    """
     appropriation_account_balances_id = models.AutoField(primary_key=True)
+    # a convenience FK back to federal accounts for ease of traversal
+    federal_account = models.ForeignKey('FederalAccount', models.DO_NOTHING, null=True)
```

Comment:
It's definitely not necessary. My thinking was that having the FK here too would reduce the traversals necessary when aggregating account balances by federal account (rather than by tas).

I'm fine removing this if we want a more pure data model.